README.md

```diff
@@ -60,13 +59,14 @@ end
 - :app_path: The Octopus Deploy Instance application directory (Defaults to C:\Octopus\Applications)
 - :trusted_cert: The Octopus Deploy Instance trusted Server cert
 - :port: The Octopus Deploy Instance port to listen on for listening tentacle (Defaults to 10933)
+- :configure_firewall: Whether cookbook will open firewall on listen tentacles (Defaults to false)
 - :polling: Whether this Octopus Deploy Instance is a polling tentacle (Defaults to False)
 - :cert_file: Where to export the Octopus Deploy Instance cert (Defaults to C:\Octopus\tentacle_cert.txt)
 - :upgrades_enabled: Whether to upgrade or downgrade the tentacle version if the windows installer version does not match what is provided in the resource. (Defaults to True)
 - :server: Url to Octopus Deploy Server (e.g https://octopus.example.com)
 - :api_key: Api Key used to register Tentacle to Octopus Server
 - :roles: Array of roles to apply to Tentacle when registering with Octopus Deploy Server (e.g ["web-server","app-server"]) 
-- :environment: Which environment the Tentacle will become part of when registering with Octopus Deploy Server
+- :environment: Which environment the Tentacle will become part of when registering with Octopus Deploy Server (Defaults to node.chef_environment )
```

Comment:
Whoops, that was supposed to be part of #45. 

I can separate them if you want.